content/integrations/entrust-keycontrol.md

```diff
@@ -0,0 +1,289 @@
+---
+title: Entrust KeyControl
+date: 2023-08-01
+lastmod: :git
+draft: false
+tableOfContents: true
+---
+
+[Entrust KeyControl](https://www.entrust.com/digital-security/key-management/keycontrol) is a proprietary KMS that provides a secret store that can be used by KES.
+
+```goat
+                +-----------------------------------------+
+ .----------.   |   .----------.    .------------------.  |
+| KES Client +--+--+ KES Server +--+ Entrust KeyControl | |
+ '----------'   |   '----------'    '------------------'  |
+                +-----------------------------------------+
+```
+
+## Prerequisites
+
+
+This guide assumes that you have setup an Entrust KeyControl v10.1 cluster. 
+
+For instructions to set up an Entrust KeyControl cluster, refer to the Entrust documentation:
+ - [Entrust KeyControl on AWS](https://docs.hytrust.com/DataControl/10.1/Online/Content/Books/Amazon-Web-Services/Creating-KC-Cluster-AWS/Configuring-the-First-KC-Node-AWS.html)
+ - [Entrust KeyControl on prem](https://docs.hytrust.com/DataControl/10.1/Online/Content/OLH-Files/Help-content-map-all-books.html) 
+
+## Set up Entrust KeyControl
+
+Before setting up the KES Server, complete the following sections in KeyControl to add a new Vault, a Box, and one or more Users.
+
+### Create a new Vault
+
+1. Login into your KeyControl cluster as `secroot` and create a new **PASM** Vault.
+2. Access Vault Management, then select **Create Vault**.
+3. Make the following entries:
+
+   - **Type**: PASM
+   - **Name**: minio
+   - **Description**: optional additional information for the vault, or leave blank
+   - **Admin Name**: user name to to manage the vault
+     
+     Note: This user has complete access to the Vault.
+   - **Admin Email**: Email address for the vault administrator
+     
+     Note: KeyControl sends a one-time password to this email address to access the vault.
+     You need this password in the next step.
+
+### Access the new Vault
+
+1. Access the Vault URL
+   
+   This URL should be in the email with the one-time password.
+   You can also find the URL from the Vault tab in KeyControl.
+2. Use the admin user name and the one-time password emailed to you to access the vault
+
+
+### Create a new Box
+
+KeyControl stores secrets in a box inside the vault.
+Add a box to store your secrets.
+
+1. From the KeyControl Secrets Vault screen, select **Manage** then **Manage Boxes**
+2. Select **Create** to add a new Box
+  
+   or
+
+   Select **Add a Box Now** from the Manage Boxes page
+4. Make the following entries ont he **About** page
```

Comment:
Same thought w.r.t making this generic, e.g. "Configure similar to the following and fill all other required fields"

Reply:
Also step 3